pwd-checker.js

```diff
@@ -1,17 +1,40 @@
-/*
+const prompt = require('prompt-sync')();
 
-Description: Develop a tool to check the strength of a password.
 
-Features:
-Prompt the user to enter a password.
-Check for:
-Minimum length.
-Presence of numbers.
-Special characters.
-Uppercase letters.
-Output a strength score or message ("Weak," "Moderate," "Strong").
-Focus: Use conditions to validate different password criteria and loops to analyze the string.
+let passwordValidation = false;
 
-*/
+while (!passwordValidation) {
+    const str = prompt('Please, enter password: ');
+
+
+    console.log('');
+
+
+    const qty = str.length;
+    const hasLowercase = /[a-z]/.test(str);
+    const hasUppercase = /[A-Z]/.test(str);
+    const hasNumber = /[0-9]/.test(str);
+    const hasOnlyNumber = /^[0-9]+$/.test(str);
+    const hasSpecialCharacter = /[!@#$%^&*(),.?":{}|<>]/.test(str);
+
```

Comment:
Mas to dobre a funguje to, skus to ale spravit za pomoci foreach cyklu, co budeme riesit na hodine 10.12.